ClientSplitBrainTest.testClientListeners_InSplitBrain

```
java.lang.AssertionError: id 0 expected:<3> but was:<0>
    at org.junit.Assert.fail(Assert.java:88)
    at org.junit.Assert.failNotEquals(Assert.java:743)
    at org.junit.Assert.assertEquals(Assert.java:118)
    at org.junit.Assert.assertEquals(Assert.java:555)
    at com.hazelcast.client.ClientSplitBrainTest$1.run(ClientSplitBrainTest.java:93)
    at com.hazelcast.test.HazelcastTestSupport.assertTrueEventually(HazelcastTestSupport.java:239)
    at com.hazelcast.client.ClientSplitBrainTest.checkEventsEventually(ClientSplitBrainTest.java:90)
    at com.hazelcast.client.ClientSplitBrainTest.testClientListeners_InSplitBrain(ClientSplitBrainTest.java:84)
```

https://hazelcast-l337.ci.cloudbees.com/job/Hazelcast-3.maintenance-OpenJDK6/com.hazelcast$hazelcast-client/83/testReport/junit/com.hazelcast.client/ClientSplitBrainTest/testClientListeners_InSplitBrain/

--- a/hazelcast-client/src/test/java/com/hazelcast/client/ClientSplitBrainTest.java
+++ b/hazelcast-client/src/test/java/com/hazelcast/client/ClientSplitBrainTest.java
@@ -92,7 +92,7 @@ public class ClientSplitBrainTest extends HazelcastTestSupport {
                 public void run() throws Exception {
                     assertEquals("id " + id, 3, atomicIntegers[id].get());
                 }
-            }, 10);
+            });
         }
     }
 

